vscode-lean4/src/leanclient.ts

```diff
@@ -253,7 +247,7 @@ export class LeanClient implements Disposable {
 
     async stop(): Promise<void> {
         assert(() => this.isStarted())
-        await this.client.stop()
+        void this.client.stop()
```

Comment:
I don't know what the vscode languageclient does when stopping.  I hope it doesn't do any cleanup, otherwise this is a race condition with the change in start.

Reply:
I don't remember exactly why I made this change, but it had something to do with startup delay. I think I wanted to rapidly restart the server for development purposes. However we don't expect anyone to have to do this, so I may well revert the change. Thoughts?

Reply:
If the only reason for the change was to make development easier, then I'd prefer to keep the `await` version.  Spawning async tasks without waiting for them feels like a race condition bound to happen.